Source/WebCore/page/Quirks.cpp

```diff
@@ -2191,6 +2196,16 @@ static void handleWarbyParkerQuirks(QuirksData& quirksData, const URL& quirksURL
 }
 #endif
 
+static void handleDailyMailCoUkQuirks(QuirksData& quirksData, const URL& quirksURL, const String& quirksDomainString, const URL& documentURL)
+{
+    if (quirksDomainString != "dailymail.co.uk"_s)
+        return;
+
+    UNUSED_PARAM(quirksURL);
+    UNUSED_PARAM(documentURL);
+    UNUSED_PARAM(quirksData);
```

Comment:
This should be setting a flag indicating that we need to do the unload. So I think you are missing a `bool` flag in QuirksData called something like, `shouldUnloadHeavyFrame` or something like that.

Reply:
done